packages/core/docs/integrate/integration-guide.md

```diff
@@ -5,319 +5,232 @@ The application does not reload automatically after saving the changes in Server
 :::
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io) server** first. We are eager to help you with building it and ensuring its high quality! Building the integration with the help of the core team is the best way to keep its quality high and making it officially recommended once completed.
+If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io) server**. We are eager to help you to ensure its high quality and maybe even officially recommend it 😉
 :::
 
 ## Introduction
 
-Integrating any eCommerce platform with Vue Storefront is a relatively simple process. The only requirement that the eCommerce platform needs to fulfill to be integrated is having a fully functional REST/GraphQL API.
+Integrating an eCommerce platform with Vue Storefront sounds scary. Luckily, some of our partners and community members with different seniority levels have successfully done it. We are sure that even without prior experience with Vue Storefront, you can too.
 
-This documentation will guide you through integrating your eCommerce platform with Vue Storefront.
+This tutorial will guide you through the process of creating integration and explain the concepts behind Vue Storefront.
```

Comment:
As it is not a tutorial per se, maybe we could look for an another word? "guide" would be an reoccurance but it might be worth looking for some synonym, I dunno.